requirements.txt

```diff
@@ -1,11 +1,11 @@
-dask>=2022.3.0
-distributed>=2022.3.0
+dask==2022.7.1
```

Comment:
Since this is going to turn into the package `install_requires`. having a range (minimum) may be preferable to the explicit pinned version here to improve the chances of finding compatibile versions of packages in a project?

Reply:
I had trouble with that because it ended up picking newer versions than we're actually compatible with, but I'm not sure where the cut-off is.